pom.xml

```diff
@@ -5,10 +5,9 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>common</artifactId>
-        <version>6.0.3</version>
+        <version>6.1.5</version>
```

Comment:
6.2.3 is latest commons,  should we upgrade to that?

Reply:
6.1.5 was the version being used by the ones wanting this CVE fixed, but I suppose we can set to latest